src/config/simple.py

```diff
@@ -0,0 +1,24 @@
+from src.data.simple import SimpleDataset
```

Comment:
Consider moving "simple" files to separate branch only with these template files